mycli/main.py

```diff
@@ -803,7 +810,9 @@ def format_output(self, title, cur, headers, expanded=False,
                 formatted = self.formatter.format_output(
                     rows, headers, format_name='vertical')
 
-            output.append(formatted)
+            if isinstance(formatted, text_type):
+                formatted = formatted.splitlines()
```

Comment:
When is `formatted` not a `text_type`? I think CLI Helpers always returns a `text_type`.